App.tsx

```diff
@@ -1,8 +1,16 @@
 import '@config/reactotron';
 import React from 'react';
 
+import { Provider } from 'react-redux';
+
 import App from './src/app';
 
+import store from '@redux/store';
```

Comment:
These imports are not correct:
The order is:
1) Global imports (`react`, `react-native`,  `react-redux`, etc...)

Maybe blank line with the new configuration 🤔 

2) Aliases

Blank line

3) Paths with `../...`

Blank line

4) Paths with `./...`

Check this please

Reply:
Cool